.github/workflows/bin-ci.yml

```diff
@@ -39,14 +55,14 @@ jobs:
         - 1.11.11+ent
     env:
       TEST_RESULTS_DIR: /tmp/test-results
-      GOTESTSUM_VERSION: 1.8.0
+      GOTESTSUM_VERSION: 1.8.2
```

Comment:
I think you need to add a dependency on test for `get-go-version` similar to what you did for lint

Reply:
Ah, yes. It actually seemed to default to go 1.18 without this. Thank you!

Reply:
I already merged the PRs for the other release branches, so I picked and push this fix to those branches:

* release/0.5.x - https://github.com/hashicorp/consul-ecs/commit/601380b2d325a826e2fdd0d9f154545685b84e9a
* main - https://github.com/hashicorp/consul-ecs/commit/473cdaf50b7ee016cd21b01c4cfb18572611a4ce